packages/date-time/src/components/Calendar/CalendarYear/CalendarYear.base.tsx

```diff
@@ -0,0 +1,458 @@
+import * as React from 'react';
+import {
+  ICalendarYearStrings,
+  ICalendarYearProps,
+  ICalendarYearRange,
+  ICalendarYearRangeToString,
+  ICalendarYear,
+  ICalendarYearHeaderProps,
+  ICalendarYearStyleProps,
+  ICalendarYearStyles
+} from './CalendarYear.types';
+import { BaseComponent, KeyCodes, getRTL, classNamesFunction, css } from 'office-ui-fabric-react/lib/Utilities';
+import { FocusZone } from 'office-ui-fabric-react/lib/FocusZone';
+import { Icon } from 'office-ui-fabric-react/lib/Icon';
+import { ICalendarIconStrings } from '../Calendar.types';
+
+const getClassNames = classNamesFunction<ICalendarYearStyleProps, ICalendarYearStyles>();
+
+const CELL_COUNT = 12;
+
+const DefaultCalendarYearStrings: ICalendarYearStrings = {
+  prevRangeAriaLabel: undefined,
+  nextRangeAriaLabel: undefined
+};
+
+const DefaultNavigationIcons: ICalendarIconStrings = {
+  leftNavigation: 'Up',
+  rightNavigation: 'Down',
+  closeIcon: 'CalculatorMultiply'
+};
+
+interface ICalendarYearGridCell {
+  focus(): void;
+}
+
+class CalendarYearGridCell extends React.Component<ICalendarYearGridCellProps, {}> implements ICalendarYearGridCell {
+  private _buttonRef: HTMLButtonElement;
+
+  public focus(): void {
+    if (this._buttonRef) {
+      this._buttonRef.focus();
+    }
+  }
+
+  public render(): JSX.Element {
+    const { styles, theme, className, highlightCurrentYear, highlightSelectedYear, year, selected, disabled, onSelectYear } = this.props;
+
+    const classNames = getClassNames(styles, {
+      theme: theme!,
+      className: className,
+      highlightCurrentYear: highlightCurrentYear,
+      highlightSelectedYear: highlightSelectedYear
+    });
+
+    return (
+      <button
+        className={css('ms-DatePicker-yearOption', classNames.yearButton, {
+          [`ms-DatePicker-day--highlighted ${classNames.selectedYear}`]: selected,
```

Comment:
ms-DatePicker-* classNames:
are we planning on bringing all of these back into date-time's Calendar components? I think I got rid of all of them before since having MergeStyles is a better way to override the styles anyway, but we can keep them if there's a reason

Reply:
No reason other than parity. Will remove.